examples/future_campaign/multi_sweep.py

```diff
@@ -150,7 +150,10 @@ def add_vax_intervention(campaign, values, min_age=0.75, max_age=15, binary_immu
     import emodpy_typhoid.interventions.typhoid_vaccine as tv
     print(f"Telling emod-api to use {manifest.schema_file} as schema.")
     campaign.set_schema(manifest.schema_file)
-    camp_coverage = values['coverage']
+    if 'coverage' in values:
```

Comment:
This is not working for run('Coverage_RIA') case.
I changed to following to replace line 153 to line 156 which seems working for all cases
```
for key in values.keys():
        if 'coverage' in key:
            camp_coverage = values[key]
            break
```

Reply:
Nice. Thanks.